src/OpenFOAM/matrices/lduMatrix/lduMatrix/lduMatrixFunctors.H

```diff
@@ -183,9 +183,125 @@ namespace Foam
             return out;
         }
     };
+    
+    
+    struct psiPlusAlphaYAFunctor
+	{
```

Comment:
Format.
This file uses mixture of tabs and spaces. Let's make it consistent with the spaces convention.

Reply:
Great.